cf-macro/index.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const _ = require('lodash')
+const Ajv = require('ajv')
+
+const alarms = require('slic-watch-core/alarms')
+const dashboard = require('slic-watch-core/dashboard')
+const CloudFormationTemplate = require('slic-watch-core/cf-template')
+const defaultConfig = require('slic-watch-core/default-config')
+const { slicWatchSchema } = require('slic-watch-core/config-schema')
+
+exports.handler = async function (event, context) {
+  const response = processFragment(event)
+  return response
+}
+
+function processFragment (event) {
+  let status = 'success'
+  let outputFragment = event.fragment
+  try {
+    const slicWatchConfig = (outputFragment.Metadata || {}).slicWatch || {}
+    const ajv = new Ajv({
+      unicodeRegExp: false
+    })
+    const config = _.merge(defaultConfig, slicWatchConfig)
+
+    const slicWatchValidate = ajv.compile(slicWatchSchema)
```

Comment:
In terms of performance, it should be better to move this line outside the handler (between lines 10 and 12).

This way, the schema is compiled only at cold start and then it can be reused across different executions of the handler.

Reply:
But slicWatchConfig is available  only in the handler to merge? Sorry, not clear what the suggestion is.